### PR TITLE
Hello Microverse Project Pull Request

### PR DESCRIPTION
**Pull request by yemidada**
Hello Microverse Project Pull Request:
In this milestone, we:
- Add an index.html file
- Add a style.css file
- Add README.md and MIT.md files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,4 +218,5 @@ Feel free to check the [issues page](../../issues/).
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (Johnadibe) [inline README.md:220]: > * [ ]  Your `MIT.md` link is not working kindly fix it. Copy the `MIT.md` file from the microverse provided [README template repo](https://github.com/microverseinc/readme-template) and paste it into your project directory
> 
> **Rename line no. 125**
> 
> ```lisp
>  This project is [MIT](./MIT.md) licensed. 
> ```

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️
### Hi @yemidada 🙋‍♂️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- Good commit history ✔️
- Good PR summary✔️

 ## Required Changes ♻️
- [ ] I noticed that your linter pass checks are not displaying in your project as stated in the project requirement. Please kindly set that up and they should all pass.😊
>> _Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git._
>
>> _Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config)._
>
- [ ] Please kindly remove the linter files from the main branch and add them to the feature branch. It is important to maintain good development practices ([GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) rules.) preventing unnecessary conflicts😊.

linter files

- [ ] Please do not open a new Pull Request for this project and submit the same for review.👍

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by yemidada** [comment]
@Johnadibe , i can see the linters display.. can you kindly confirm it before i submit

File: index.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Hello Microverse!" />
+    <link href="style.css" rel="stylesheet">
+    <title>Hello Microverse!</title>
+</head>
+
+<body>
+
+    <h1>Hello Microverse!</h1>
+
```
Comment (ademibowale) [inline index.html:16]: - [ ] [Optional] I would suggest you to use class name for the `<h1>` tag element, CSS classes enable you to apply unique style properties to groups of HTML elements to achieve your desired web page appearance also in another world  HTML class attribute specifies a class name. You can use it to select and access HTML elements for various tasks performed using CSS .. e.g ` <h1 class="header">Hello Microverse</h1>`

**Review by ademibowale — APPROVED** [review]
Hi @yemidada  , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**